.circleci/config.yml

```diff
@@ -544,6 +544,26 @@ jobs:
       - run: git diff-index --quiet HEAD
       - magma_slack_notify
 
+  # Fail if terraform variables, helm charts and orcl is out of sync
+  deploy-sync-checkin:
+    machine:
+      image: ubuntu-1604:201903-01
+      #docker_layer_caching: true
+    environment:
+      MAGMA_ROOT: /home/circleci/project
+    steps:
+      - checkout
+      - build/determinator:
+          <<: *all_gateways_build_verify
+      - docker/install-dc
+      - run: sudo apt-get update
```

Comment:
I think this is unnecessary, both here and the other 1-2 places where it seems to be used for no purpose